Qpf/Macro/Data/Constructors.lean

```diff
@@ -17,9 +18,13 @@ partial def countConstructorArgs : Syntax → Nat
   Add definitions for constructors
   that are generic across two input types shape and name.
   Additionally we allow the user to control how names are generated.
+  The binders param allows for adding constant binders to an expression.
 -/
-def mkConstructorsWithNameAndType (view : DataView) (shape : Name)
-    (nameGen : CtorView → Name) (ty : Term) : CommandElabM Unit := do
+def mkConstructorsWithNameAndType
+    (view : DataView) (shape : Name)
+    (nameGen : CtorView → Name) (argTy retTy : Term)
+    (binders : TSyntaxArray ``Parser.Term.bracketedBinder)
```

Comment:
```suggestion
    (binders : TSyntaxArray ``Parser.Term.bracketedBinder := #[])
```
This seems like a good place to use a default value (but feel free to ignore if you disagree)